### PR TITLE
Readme fixes

### PR DESCRIPTION
**Pull request by salarcon215**
Hi Tor team,

In preparation for use on Measurementlab.net, I made some changes to the installation instructions in the readme. I took out references to outdated code based on feedback from art@torproject.org, and made fixes and readability changes based on my own testing. Note that everything below "Using Ooniprobe" is mostly untouched.

I'll look out for any comments or corrections and happily incorporate them.  Thanks!

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -194,6 +232,19 @@ with ``ooniresources``::
 Generating decks
 ----------------
 
+Try out decks of tests
+----------------------
```
Comment (anarcat) [inline README.rst]: this seems to be misordered...

File: README.rst [hunk 2]
```diff
@@ -46,107 +46,135 @@ probe is a liability for you, please be aware of this risk.
...
+        brew install git
 
-On debian/ubuntu::
+    b) On Debian/Ubuntu::
 
-    sudo apt-get install git
+        sudo apt-get install git
 
-3. Open a Terminal and run::
+2) Open a Terminal and run::
```
Comment (anarcat) [inline README.rst]: not sure this markup will work in rst...

**Comment by hellais** [comment]
Sorry for being a pain, can you folks rebase this against the current master?

**Comment by salarcon215** [comment]
Hey Arturo, will do. I also sent your requests about mlab-ns to the people who can respond accurately.

**Comment by anadahz** [comment]
Most (if not all) of the documentation updates proposed are obsolete since the documentation has significantly changed and is not relevant to the current.
I propose that we should close this PR.

**Comment by salarcon215** [comment]
Sounds good. Sorry it languished.

**Pull request closed without merging**